### ISSUE TITLE
Network Slicing Engine

### ISSUE THREAD

**Issue by noormohammedli** [openbaton/bootstrap#29]
Hello Sir, 
How could i run the Network Slicing Engine? 
after i get the bootstrap from Github, I run the following command to run the docker-compose file to build the NSE. 
$ env HOST_IP=$YOUR_LOCAL_IP docker-compose -f docker-compose-full.yml up -d
then i run the " $ until curl -sSf --stderr /dev/null http://localhost:8080; do printf '.' && sleep 5;done "

but I could not see the NSE when i display all docker container "sudo docker-compose ps"

I tried this comment as well but i could not access the NSE "$ env HOST_IP=$YOUR_LOCAL_IP docker-compose -f docker-compose-nse.yml up -d"

I read the documentation on " https://openbaton.github.io/documentation/network-slicing/ "
and "https://github.com/openbaton/bootstrap"

I could not access the Once the access the screen session after the NSE is started. 

Basically, I trying to create 3 slices type on top of the open-baton and do autoscaling for the slices. 
Could you please help me with that? is there any video or example explain the slice creation with open-baton?

Thanks
